docs/reference/rest_api.md

```diff
@@ -1969,3 +1969,217 @@ Content-Type: application/json
   "links": { ... }
 }
 ```
+
+    
+## Communities (Preview)
+    
+### Create a Community
+
+`POST /api/communities/<comid>/`
+    
+**Parameters**
+
+| Name       | Type   | Location | Description                                                  |
+| ---------- | ------ | -------- | ------------------------------------------------------------ |
+| `comid`       | string | path     | Identifier of the community, e.g.  `my_community`                |
+| `accept`   | string | header   | - `application/json` (default)<br />- `application/vnd.inveniordm.v1+json` |
+
+**Request**
+
+```http
+POST /api/communities/<comid>/ HTTP/1.1
+Accept: application/json
+    
+{
+  "title": "My Community",
+  "description": "This is an example Community.",
+  "type": "event",
+  "alternate_identifiers": [{"scheme": "ror"}, {"identifier": "03yrm5c26"}],
+  "curation_policy": "This is the kind of records we accept.",
+  "page": "Information for my community.",
+  "website": "https://www.my_community.org",
+  "domains": ["biology", "chemistry"],
+  "funding": ["Grant-000000"],
+  "visibility": "public",
+  "member_policy": "open",
+  "record_policy": "open"
```

Comment:
I guess we could also start with a "Community metadata" section in the metadata reference, so that we can go a bit more into detail of the values each field can take.